tedana/io.py

```diff
@@ -17,6 +16,133 @@
 RepLGR = logging.getLogger('REPORT')
 RefLGR = logging.getLogger('REFERENCES')
 
+outdir = '.'
+prefix = ''
+convention = 'bids'
```

Comment:
These aren't going to override user input are they? I've looked around and this pull request is definitely what I was imagining. And bonus, if some new naming convention takes the world by storm - just add another set at the end and index with [2]!

Reply:
No, this is a just a default. In the main workflow it's specified with `--convention`, and in the API it's specified with `convention=`

Reply:
> if some new naming convention takes the world by storm - just add another set at the end and index with [2]
That's what I was thinking. Plus at some point there will be BIDSv2.0 anyway. So these convention changes are probably inevitable.
If we want to get fancy we can set things up so that any given file has a full description and some other features, but I think this is an okay start.

Reply:
The same line confused me. This might be a good place for a comment to note `convention = 'bids'` sets the default, which is typically changed when tedana.py is run.